snd-server/migrations/000_create_users.sql

```diff
@@ -1,5 +1,6 @@
 create table if not exists users (
   id integer primary key,
+  username text not null,
   email text not null,
-  access_key_hash text
+  hashed_password text
 );
```

Comment:
Note that you'll have to manually drop current db file on the server before next deployment if you want to use the same migration file - for updates to kick in.

Otherwise you'll have to create next migration script in e.g. `001_update_users.sql` with `alter table .. add column` query, but there are [some limitations](https://www.sqlite.org/lang_altertable.html) when adding new columns